crates/nu-utils/src/default_files/sample_config.nu

```diff
@@ -727,20 +730,19 @@ $env.config.color_config.int = {||
 # Additional type values (without examples):
 $env.config.color_config.string      # String
 $env.config.color_config.float       # Float value
-$env.config.color_config.glob        # Glob value (must be declared)
+# TODO: https://github.com/nushell/nushell/issues/14600
+#$env.config.color_config.glob        # Glob value (must be declared)
```

Comment:
Probably other deleted options in this list are affected by the same linked issue

Reply:
I think these are fine.  They are simply lists of settings here.  We don't need to show a "default" or example value for each one, IMHO.  The code is uncommented so that it syntax highlighted when viewing through `nu-highlight`. Ultimately, this is "documentation" that probably gets moved over to the Book itself, and these files go away.

Reply:
Also, this kind of approach (storing documentation as default config) is not really viable for something like nixos (where configs are managed by nix itself, you don't have a default config in your disposal)

Reply:
@PerchunPak Agreed - That's why the only method for a *user* to access them is `nu config --sample`.  The end-user (and package manager) never has access to them directly.

In the past (0.100 and before) doc and defaults were mixed.  Starting in 0.101 with these changes, they no longer are.

Reply:
So should I return all other deleted options?

Reply:
Yes, any option that is just listed by itself without an assignment should be returned. Thanks!